bin/template/package.json

```diff
@@ -8,28 +8,36 @@
     "start": "nodemon make.js run",
     "build": "node make.js",
     "deploy": "node make.js deploy",
+    "deploy-website": "node make.js deploy-website",
+    "deploy-hosted-zone": "node make.js deploy-hosted-zone",
     "delete": "node make.js delete",
     "test": "mocha tests/**/*.js -R spec"
   },
   "dependencies": {
-    "aws-sdk": "^2.221.1",
-    "axios": "^0.17.1",
-    "ci-build-tools": ">=1.0",
-    "fs-extra": "^6.0.1",
-    "jsonwebtoken": "^8.1.0",
-    "jwk-to-pem": "^1.2.6",
-    "microservice-utilities": "^0.1",
-    "openapi-factory": "^4.3"
+    "axios": "^0.18.0",
+    "ci-build-tools": "^1.0.13",
+    "fs-extra": "^7.0.1",
+    "jsonwebtoken": "^8.4.0",
+    "jwk-to-pem": "^2.0.0",
+    "microservice-utilities": "^0.1.78",
+    "openapi-factory": "^4.3.178"
```

Comment:
If we could leave this as `^4.3` unless there was a specific reason to change.  My goal was for new users to automatically be able to get the latest version without having to constantly come back here to update it.